### PR TITLE
Add docs for new Unsubscribe metrics

### PR DESCRIPTION
**Pull request by tough-griff**
😄

### CODE REVIEW

File: articles/tech-resources/sparkpost-event-metrics-definitions.md [hunk 1]
```diff
@@ -32,56 +32,57 @@ Individual events are provided via the webhook data stream and can be retrieved
...
+| Unique Confirmed Opens           | Total number of messages that were either rendered or had at least one link selected. (This metric may take longer to display.)               | Sum (unique messages opened or clicked)                            | The unique number of **unique messages** opened (whether fully opened or within the preview pane) and recorded using a tracking pixel OR if images are blocked and the user clicks any link including the unsubscribe link. Includes Opened OR Clicked unique messages.                                                                                                                                                                                 |
+| Open Rate (approx.)              | Open Rate using the Unique Confirmed Opens (approx.) metric                                                                                   | Percentage (Unique Confirmed Opens (ap
```
Comment (tough-griff) [inline articles/tech-resources/sparkpost-event-metrics-definitions.md]: This and below are the only new lines, others were table format fixes.

Comment (tough-griff) [inline articles/tech-resources/sparkpost-event-metrics-definitions.md]: This and below are the only new lines, others were table format fixes.

File: articles/tech-resources/sparkpost-event-metrics-definitions.md [hunk 2]
```diff
@@ -32,56 +32,57 @@ Individual events are provided via the webhook data stream and can be retrieved
...
+| Unique Confirmed Opens           | Total number of messages that were either rendered or had at least one link selected. (This metric may take longer to display.)             | Sum (unique messages opened or clicked)                            | The unique number of **unique messages** opened (whether fully opened or within the preview pane) and recorded using a tracking pixel OR if images are blocked and the user clicks any link including the unsubscribe link. Includes Opened OR Clicked unique messages.                                                                                                                                                                                 |
+| Open Rate (approx.)              | Open Rate using the Unique Confirmed Opens (approx.) metric                                                                                 | Percentage (Unique Confirmed Opens (approx.) / Accepted)           |
```
Comment (tough-griff) [inline articles/tech-resources/sparkpost-event-metrics-definitions.md]: This and the following are the only new lines, others were table format fixes. (Prettier handled that for me)

Comment (tough-griff) [inline articles/tech-resources/sparkpost-event-metrics-definitions.md]: Here's what matters.

File: articles/reporting/metrics-definitions.md [hunk 3]
```diff
@@ -159,3 +159,11 @@ Number of messages opened or have at least one link clicked.
...
 
 Number of unique recipients who open a message.
+
+**Unsubscribes**
+
+Total number of times a recipient clicked a specifically tagged unsubscribe link or unsubscribed by using the ISP list unsubscribe feature.
+
+**Unsubscribe Rate**
+
+The percentage messages that resulted in unsubscribes. Calculated by _number of all unsubscribes / number of messages sent._
```
Comment (bkemper) [inline articles/reporting/metrics-definitions.md]: number of messages sent or accepted?

Comment (tough-griff) [reply articles/reporting/metrics-definitions.md]: Accepted, but I didn't see that in any of the existing `Calculated by` notes. Used sent for consistency. However, some rates are missing the calculation alltogether...

File: articles/reporting/metrics-definitions.md [hunk 4]
```diff
@@ -110,7 +110,7 @@ Number of messages for which SparkPost attempts delivery.  This is the number o
 
 **Open Rate**
 
-Percentage of “accepted” messages that are either rendered or have at least one click.
+Percentage of “accepted” messages that are either rendered or have at least one click. Calculated by _number of messages opened / number of messages delivered._
```
Comment (bkemper) [inline articles/reporting/metrics-definitions.md]: number of messages ~~delivered~~ accepted

File: articles/reporting/metrics-definitions.md [hunk 5]
```diff
@@ -126,27 +126,27 @@ Number of messages for which SparkPost attempts delivery.  This is the number o
...
+The percentage of messages that are "soft" bounced.  Calculated by _number of soft bounces / number of messages sent._
 
 **Soft Bounced**
 
 Number of messages that are "soft" bounced by the recipient's mailbox provider.  (A soft bounce occurs when recipient's mailbox provider states that message cannot be accepted at this time, common example: mailbox full.)
 
 **Spam Complaint Rate**
 
-Percentage of “accepted” messages that have a spam complaint.
+Percentage of “accepted” messages that have a spam complaint. Calculated by _number of spam complaints / number of messages delivered._
```
Comment (bkemper) [inline articles/reporting/metrics-definitions.md]: number of messages ~~delivered~~ accepted

File: articles/reporting/metrics-definitions.md [hunk 6]
```diff
@@ -159,3 +159,11 @@ Number of messages opened or have at least one link clicked.
...
 
 Number of unique recipients who open a message.
+
+**Unsubscribes**
+
+Total number of times a recipient clicked a specifically tagged unsubscribe link or unsubscribed by using the ISP list unsubscribe feature.
+
+**Unsubscribe Rate**
+
+The percentage messages that resulted in unsubscribes. Calculated by _number of all unsubscribes / number of messages delivered._
```
Comment (bkemper) [inline articles/reporting/metrics-definitions.md]: percentage **of** messages

number of messages ~~delivered~~ accepted

**Pull request merged**